.eslintrc.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
 	"root": true,
 	"env": {
-		"es6": true
+		"es6": true,
+		"browser": true
```

Comment:
I don't think it's a good idea to add `browser` environment to `/.eslintrc.js`, in which case eslint won't be able to detect errors if we have wrongly used browser APIs in `tools/`.

The solution is to use multi `.eslintrc.js` files (see https://github.com/to2mbn/skinview3d/commit/79167ab0f855beb6942d576735a19d65f0f503e2).

Reply:
Agree.